trip-ninja-docs/src/index.css

```diff
@@ -36,13 +37,18 @@ h3 {
   font-weight: 600;
 }
 
+a:hover {
```

Comment:
lolo i was setting this by the css class, this makes more sense! will refactor mine when this is merged 🙌